presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -63,8 +67,9 @@
     private RegexLibrary regexLibrary = JONI;
     private boolean spillEnabled;
     private DataSize operatorMemoryLimitBeforeSpill = new DataSize(4, DataSize.Unit.MEGABYTE);
-    private Path spillerSpillPath = Paths.get(System.getProperty("java.io.tmpdir"), "presto", "spills");
+    private List<Path> spillerSpillPaths = ImmutableList.of(Paths.get(System.getProperty("java.io.tmpdir"), "presto", "spills"));
     private int spillerThreads = 4;
+    private DataSize spillMinimumFreeSpaceThreshold = DataSize.succinctDataSize(1, DataSize.Unit.GIGABYTE);
```

Comment:
Well actually I cannot change that. It seems that there is no mechanism for configs depending on other configs. @Inject does not work.